pandas/core/groupby/generic.py

```diff
@@ -1001,14 +1001,65 @@ def take(
         result = self._op_via_apply("take", indices=indices, axis=axis, **kwargs)
         return result
 
-    @doc(Series.skew.__doc__)
     def skew(
         self,
         axis: Axis | lib.NoDefault = lib.no_default,
         skipna: bool = True,
         numeric_only: bool = False,
         **kwargs,
     ) -> Series:
+        """
+        Return unbiased skew within groups.
+
+        Normalized by N-1.
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns', None}, default 0
+            Axis for the function to be applied on.
+            For `Series` this parameter is unused and defaults to 0.
```

Comment:
For reST, there should be two backticks, ` ``Series`` ` (markdown would only have one). This phrasing is used when the docstring is shared between Series and DataFrame implementations. Since this is just for Series, what do you think about something like:

> This parameter is only for compatibility with DataFrame and is unused.

Reply:
Maybe it makes sense to write the same for `numeric_only`?

Reply:
I think numeric_only is slightly different - if you specify `numeric_only=True` and the Series is non-numeric, then it will raise. This could be useful to identify errors - similar to adding an assert.